frontend/src/metabase/account/notifications/components/HelpModal/HelpModal.unit.spec.js

```diff
@@ -1,12 +1,16 @@
-import { render, screen } from "@testing-library/react";
 import { mockSettings } from "__support__/settings";
+import { renderWithProviders, screen } from "__support__/ui";
 import HelpModal from "./HelpModal";
 
+function setup({ adminEmail, onClose } = {}) {
+  renderWithProviders(<HelpModal adminEmail={adminEmail} onClose={onClose} />);
```

Comment:
you'll see other tests have very similar changes as well, the point is we change some components to get the application name from Redux store, and their respective unit tests still rely on using `render` which doesn't include Redux, so I need to change from `render` to `renderWithProviders`